tasks/configure.yml

```diff
@@ -1,10 +1,11 @@
 ---
 
-- name: Set hostname
+- name: Ensure hostname is resolvable
   become: yes
   lineinfile:
     dest: /etc/hosts
-    line: "127.0.0.1 {{ ansible_hostname }}"
+    line: "127.0.0.1 localhost {{ ansible_hostname }}"
```

Comment:
change it to what? (to localhost?)

Problems here is that Go CD can't resolve `{{ ansible_hostname }}` on start.